MANIFEST.in

```diff
@@ -1,9 +1,10 @@
-include apache-2.0.LICENSE
+include LICENSE
 include NOTICE
 include manage.py
 include MANIFEST.in
 include Makefile
 include *.rst
+include scan.NOTICE
```

Comment:
I do not this will have any effect since "scan.NOTICE" is a package file and not at the root?
So IMHO you can remove that.
```suggestion
```